src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,56 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, V> {
+    private T first;
+    private V second;
 
+    public Pair(T first, V second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public T getFirst() {
+        return first;
+    }
+
+    public V getSecond() {
+        return second;
+    }
+
+    public static <T, V> Pair<T, V> of(T first, V second) {
+        Pair<T, V> pair = new Pair<T, V>(first, second);
+        pair.first = first;
+        pair.second = second;
```

Comment:
there is no need to set fields, you just return the object that you get from the constructor